### PR TITLE
Initial version release

### PR DESCRIPTION
**Pull request by greermichael**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  type        = "list"
+}
+
+variable "domain_name" {
+  description = "Name of the Elasticsearch cluster"
+}
+
+variable "elasticsearch_version" {
+  description = "Verion of Elasticsearch cluster"
+  default     = "6.4"
```
Comment (radepal) [inline variables.tf]: 6.5

Comment (greermichael) [reply variables.tf]: done

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+}
+
+variable "dedicated_master_type" {
+  description = "Instance type of the elasticsearch dedicated masters"
+  default     = "r5.large.elasticsearch"
+}
+
+variable "elasticsearch_allowed_cidrs" {
+  description = "List of CIDR ranges to allow HTTPS access to the elasticsearch cluster"
+  default     = []
```
Comment (radepal) [inline variables.tf]:
```suggestion
    type        = "list"
    default     = []
```

Comment (greermichael) [reply variables.tf]: fixed

File: README.md [hunk 3]
```diff
@@ -1,15 +1,60 @@
...
+| elasticsearch_logstash_prefix | Prefix of logstash indexes created in ElasticSearch | string | `logstash` | no |
+| elasticsearch_version | Verion of Elasticsearch cluster | string | `6.5` | no |
+| elasticsearch_volume_size | The size of the EBS volumes attached in GB (size per instance) | string | `10` | no |
+| enable_fluentd | Enable/Deploy fluentd into the cluster and ship logs to ES | string | `true` | no |
+| fluentd_image_repository | The URI of the repository containing the fluentd image | string | `` | no |
+| fluentd_image_tag | The tag of the image to use for fluentd | string | `latest` | no |
+| fluentd_namespace | The kubernetes namespace to use for the fluentd deployment | string | `logging` | no |
+| fluentd_release_name | Name for the fluentd deployment | string | `fluentd-elasticsearch` | no |
+| region | AWS region name | string | - | yes |
+| subnet_ids | List of subnet ids to host the Elasticsearch cluster | list | - | yes |
```
[the remark this answers is not in the document]
Comment (greermichael) [reply README.md]: done

File: variables.tf [hunk 4]
```diff
@@ -0,0 +1,109 @@
+variable "region" {
+  description = "AWS region name"
+}
+
+variable "vpc_id" {
+  description = "VPC ID"
+}
+
+variable "subnet_ids" {
```
[the remark this answers is not in the document]
Comment (greermichael) [reply variables.tf]: done

**Pull request merged**